### PR TITLE
Rename the OSG section to dHTC

### PR DESCRIPTION
**Pull request by brianhlin**
I thought OSG is a little confusing here since ostensibly all sections are "OSG" since it's part of the OSG Virtual School. Alternatively I could just rename the section in the Overview page `Running in the OSG` and keep all the exercises named `OSG Exercise N`

Thoughts @lmichael107?

### CODE REVIEW

**Comment by osg-cat** [comment]
Maybe “Grid”? I mean, “HTC” and “dHTC” are a little hard to keep straight, too…

**Comment by brianhlin** [comment]
Done and ready for review

File: docs/materials/osg/ex2-login-scp.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ tarballs) whose usage is as follows:
         :::console
         user@learn $ tar -tzvf <archive filename>
 
-Using the above knowledge, log into `learn.chtc.wisc.edu`, create a tarball that contains the OSG exercise 1 directory,
+Using the above knowledge, log into `learn.chtc.wisc.edu`, create a tarball that contains the dHTC exercise 1 directory,
```
Comment (osg-cat) [inline docs/materials/osg/ex2-login-scp.md]: Should this be “Grid”, too?

File: docs/materials/osg/ex2-login-scp.md [hunk 2]
```diff
@@ -120,7 +120,7 @@ server's operating system:
 #### Mac and Linux users
 
 `scp` should be included by default and available via the terminal on both Mac and Linux operating systems.
-Open a terminal window on your laptop and try copying the tarball containing the OSG exercise 1 from
+Open a terminal window on your laptop and try copying the tarball containing the dHTC exercise 1 from
```
Comment (osg-cat) [inline docs/materials/osg/ex2-login-scp.md]: Ditto.

File: docs/materials/osg/ex2-login-scp.md [hunk 3]
```diff
@@ -129,7 +129,7 @@ WinSCP is an `scp` client for Windows operating systems.
 
 1.  Install WinSCP from <https://winscp.net/eng/index.php>
 1.  Start WinSCP and enter your SSH credentials for `login04.osgconnect.net`
-1.  Copy the tarball containing OSG exercise 1 to your laptop
+1.  Copy the tarball containing dHTC exercise 1 to your laptop
```
Comment (osg-cat) [inline docs/materials/osg/ex2-login-scp.md]: Ditto.

File: docs/materials/osg/ex2-login-scp.md [hunk 4]
```diff
@@ -150,9 +150,9 @@ Both of these feature are helpful when you're having connectivity issues so that
 from scratch every time your connection fails.
 
 1.  Log in to `login04.osgconnect.net`
-1.  Use `rsync` to transfer the folder containing OSG exercise 1 on `learn.chtc.wisc.edu` to `login04.osgconnect.net`
+1.  Use `rsync` to transfer the folder containing dHTC exercise 1 on `learn.chtc.wisc.edu` to `login04.osgconnect.net`
```
Comment (osg-cat) [inline docs/materials/osg/ex2-login-scp.md]: Ditto.

File: docs/materials/osg/ex2-login-scp.md [hunk 5]
```diff
@@ -150,9 +150,9 @@ Both of these feature are helpful when you're having connectivity issues so that
...
 1.  In a separate terminal window, log in to `learn.chtc.wisc.edu`
-1.  Create a new file in your OSG exercise 1 folder on `learn.chtc.wisc.edu` with the `touch` command:
+1.  Create a new file in your dHTC exercise 1 folder on `learn.chtc.wisc.edu` with the `touch` command:
```
Comment (osg-cat) [inline docs/materials/osg/ex2-login-scp.md]: Ditto.

File: docs/materials/osg/ex3-submit-osg.md [hunk 6]
```diff
@@ -12,17 +12,17 @@ Where in the world are my jobs? (Part 2)
 
 In this version of the geolocating exercise, you will submit jobs to the OSG from `login04.osgconnect.net` and
 hopefully getting back much more interesting results!
-You will be using the same exact payload as you did in [OSG exercise 1](/materials/osg/ex1-submit-refresher).
+You will be using the same exact payload as you did in [dHTC exercise 1](/materials/osg/ex1-submit-refresher).
```
Comment (osg-cat) [inline docs/materials/osg/ex3-submit-osg.md]: Ditto.

File: docs/materials/osg/ex3-submit-osg.md [hunk 7]
```diff
@@ -12,17 +12,17 @@ Where in the world are my jobs? (Part 2)
...
 
 Now to create submit files that will run in the OSG!
 
 1. If not already logged in, `ssh` into `login04.osgconnect.net`
-1. Make a new directory for this exercise, `osg-ex3` and change into it
-1. Use `scp` or `rsync` from [OSG exercise 2](/materials/osg/ex2-login-scp) to copy over the executable and input
-   file from the `osg-ex1` directory from `learn`.
-1. Re-create the submit file from OSG exercise 1 except this time around change your submit file so that it submits
+1. Make a new directory for this exercise, `grid-ex3` and change into it
+1. Use `scp` or `rsync` from [dHTC exercise 2](/materials/osg/ex2-login-scp) to copy over the executable and input
```
Comment (osg-cat) [inline docs/materials/osg/ex3-submit-osg.md]: Ditto.

File: docs/materials/osg/ex3-submit-osg.md [hunk 8]
```diff
@@ -12,17 +12,17 @@ Where in the world are my jobs? (Part 2)
...
+   file from the `grid-ex1` directory from `learn`.
+1. Re-create the submit file from dHTC exercise 1 except this time around change your submit file so that it submits
```
Comment (osg-cat) [inline docs/materials/osg/ex3-submit-osg.md]: Ditto.

File: docs/materials/osg/ex4-hardware-diffs.md [hunk 9]
```diff
@@ -101,7 +101,7 @@ Checking OSG memory availability
 
 For the second part of the exercise, you will just copy over the directory from the [above section](#checking-chtc-memory-availability)
 on `learn.chtc.wisc.edu` to `login04.osgconnect.net` and resubmit your jobs to the OSG.
-If you get stuck during the copying process, refer to [OSG exercise 4](/materials/osg/ex2-login-scp.md).
+If you get stuck during the copying process, refer to [OSG exercise 2](/materials/osg/ex2-login-scp.md).
```
Comment (osg-cat) [inline docs/materials/osg/ex4-hardware-diffs.md]: Should “OSG” be “Grid”?

**Review by osg-cat — COMMENTED** [review]
Questions in the review. Also, do you want to change the directory name to `grid`?

**Comment by brianhlin** [comment]
Find and replace fail! Should be fixed now

**Pull request merged**